captcha-validation-service/src/main/resources/bootstrap.properties

```diff
@@ -0,0 +1,31 @@
+server.port=9089
+server.servlet.context-path=/esignet/v1/captcha
+spring.cloud.config.uri=localhost
+spring.cloud.config.label=master
+spring.cloud.config.name=esignet
+spring.application.name=captcha-validation-service
+spring.profiles.active=dev
+management.endpoint.health.show-details=always
+management.endpoints.web.exposure.include=info,health,refresh
+
+springdoc.paths-to-exclude=/swagger-resources/**
+security.basic.enabled=false
+
+mosip.esignet.captcha.sitekey=6Lc4pewUAAAAANr3JkwuSVNCzQi9Z31g-NVKMXBX
```

Comment:
From line 14-17, these properties should not be part of bootstrap.properties, instead, they should be part of application-dev.properties(as your active profile is set to dev)